app/Family.php

```diff
@@ -279,4 +279,15 @@ public function initialCentre()
     {
         return $this->belongsTo('App\Centre', 'initial_centre_id');
     }
+
+    public function scopeWithPrimaryCarer($query)
+    {
+        $subQuery = \DB::table('carers')
+            ->select('name')
+            ->whereRaw('family_id = families.id')
+            ->oldest()
```

Comment:
It's always the first input? And can not be amended?

Reply:
Will check...